lib/proxy.js

```diff
@@ -75,13 +75,21 @@ module.exports = function(options) {
 
     // If the req has a body, pipe it into the proxy request
     var apiRequest;
+    var originalApiRequest;
     if (is.hasBody(req)) {
       debug('piping req body to remote http endpoint');
-      apiRequest = req.pipe(request(apiRequestOptions));
+      originalApiRequest = request(apiRequestOptions);
```

Comment:
I'm a little unclear what is going on here. Just want to make sure there aren't 2 different network requests being spawned.